algorithms/hhl/hhl/hhl.ipynb

```diff
@@ -1,982 +1,1292 @@
 {
```

Comment:
<p>"The built-in function <code>matrix_to_hamiltonian</code> encodes the unitary matrix U... Pauli strings": the sentence is a bit unclear.</p><p>Suggested modification:</p><p>"The built-in function <code>matrix_to_hamiltonian</code> encodes the matrix A into a sum of Pauli strings, which then allows to encode the unitary matrix U with a product formula (Suzuki-Trotter)"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Clarified.</p>